packages/mds-web-sockets/server.ts

```diff
@@ -1,14 +1,19 @@
 import logger from '@mds-core/mds-logger'
 import { seconds, getEnvVar } from '@mds-core/mds-utils'
+import { Json } from '@mds-core/mds-types'
 import WebSocket from 'ws'
 import { setWsHeartbeat } from 'ws-heartbeat/server'
-import { Telemetry, VehicleEvent } from '@mds-core/mds-types'
 import { ApiServer, HttpServer } from '@mds-core/mds-api-server'
 import { initializeNatsSubscriber } from '@mds-core/mds-stream/nats/nats'
 import { Clients } from './clients'
-import { ENTITY_TYPE } from './types'
-
-export const WebSocketServer = () => {
+import { EntityTypes } from './types'
+
+/**
+ * Web Socket Server that pas
+ * @param entityTypes - entities to pass on to clients
+ */
+export const WebSocketServer = <T extends readonly string[]>(entityTypes?: T) => {
+  const supportedEntities = entityTypes || EntityTypes
```

Comment:
I tried putting EntityTypes as the default value in the signature and it gave a TS error on that stating as follows, so I simply moved the assignment down into the body:

`
Type 'readonly ["event", "telemetry"]' is not assignable to type 'T'.
  'readonly ["event", "telemetry"]' is assignable to the constraint of type 'T', but 'T' could be instantiated with a different subtype of constraint 'readonly string[]
`

Reply:
Did a bit of digging and can't seem to think of a better solution than what we have here, I don't really like that the type of `supportedEntities` is a `T | readonly ['events', 'telemetries']` though :/